tools/stress-cluster/cluster/kubernetes/stress-infrastructure/templates/stresswatcher.yaml

```diff
@@ -40,12 +40,12 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: stress-watcher-cluster-role-binding
+  name: stress-writer-{{ .Release.Namespace }}
```

Comment:
The naming seems to be misleading? Or is this specific naming needed here?

Reply:
I agree it's not the best name. Any suggestions you can think of? Perhaps like `stress-resource-owner` or something?

Reply:
Yeah, I think stress-resource-owner fits the context

Reply:
Thinking a little more, how about `stress-resource-patcher`? It's not a full owner since it can't delete, and the intent is to make small patches to resources.

Reply:
Ok, sgtm